src/pygeoml200/calibration.py

```diff
@@ -47,13 +47,22 @@ def place_calibration_system(b: core.InstrumentationData) -> None:
     calib_tube_length = []
     calib_tube_xy = np.empty((2, 4))
 
-    sis_cfg = b.runtime_config.get("sis", {})
+    sis_cfg_raw = b.runtime_config.get("sis", {})
+    # normalize SIS config keys to integers to accept both "1" and 1 keys
+    sis_cfg: dict[int, AttrsDict | dict | None] = {}
+    if hasattr(sis_cfg_raw, "items"):
+        for k, v in sis_cfg_raw.items():
+            try:
+                sis_cfg[int(k)] = v
+            except (TypeError, ValueError):
```

Comment:
I would not do this, this is discarding config options from the user silently if they make a mistake. I would just remove the try/except block and let the user deal with the error when they have a wrong config? (we certainly do this quite often in other places)